cosmos/lib/address.go

```diff
@@ -31,6 +31,12 @@ func AccAddressToEthAddress(accAddress sdk.AccAddress) common.Address {
 	return common.BytesToAddress(accAddress)
 }
 
+// EthAddressFromBEch32 converts a Bech32 string to an Ethereum `Address`.
+func EthAddressFromBech32(bech32Str string) common.Address {
```

Comment:
`MustEthAddressFromBech32` and then panic if line 36 has an error != nil ?

Reply:
nah mempool shouldn't crash